src/core/coreav.cpp

```diff
@@ -31,7 +31,52 @@
 #include <QCoreApplication>
 #include <QtConcurrent/QtConcurrentRun>
 
+/**
+@fn void CoreAV::avInvite(uint32_t friendId, bool video)
+@brief Sent when a friend calls us.
+@param friendId Id of friend in call list.
+@param video Does't this invite in video chat?
```

Comment:
If you get invited in chat with video, `video` must be true.
If you get invited in audio-only chat, `video` must be false.
What I should write?